### PR TITLE
feat: Dex history page

### LINKED ISSUE #9: Implement DEX swapper history page
Implement a pixel perfect DEX swapper history page from the [FIGMA](https://www.figma.com/design/JW1DKwDBVJq97whSwlWDLS/Auto-swapper?node-id=282-1715&node-type=frame&t=tMxPcYhV9Bd5SNnM-0) design.

### PR DESCRIPTION
**Pull request by Yunusabdul38**
## Description

_Detailed description of proposed changes_

## Related Issue(s)

Closes #9
## Checklist:

- [x] Read the [contributing docs](../CONTRIBUTING.md) (if this is your first contribution)
- [x] Verified this is not a duplicate of any existing pull request

- [x] Performed a self-review of the code
- [x] Rebased to the last commit of the target branch (or merged it into my branch)
- [x] Linked the issues which this PR resolves
- [x] Documented the changes in code (API docs will be generated automatically)

### CODE REVIEW

**Comment by Yunusabdul38** [comment]
@Oshioke-Salaki pls review

File: app/history/page.tsx
Comment (Oshioke-Salaki) [inline app/history/page.tsx:1]: This doesn't take up the full height of the display screen.
Screenshot 2024-11-28 at 10 07 07

**Review by Oshioke-Salaki — CHANGES_REQUESTED** [review]

File: app/history/page.tsx
Comment (Oshioke-Salaki) [inline app/history/page.tsx:1]: Screenshot 2024-11-28 at 11 25 18
Align the items in the `to` column to the left

File: app/history/page.tsx [hunk 1]
```diff
@@ -0,0 +1,241 @@
+import btc from "../../public/coin-logos/btc-logo.png"
```
Comment (Oshioke-Salaki) [inline app/history/page.tsx:1]: snyc your fork and make use of the svg versions of the coins. If a particular svg coin is not available in the repo, you can go ahead and export it from the figma file and make use of it.

**Review by Oshioke-Salaki — CHANGES_REQUESTED** [review]

**Comment by Yunusabdul38** [comment]
@Oshioke-Salaki pls review

**Comment by Oshioke-Salaki** [comment]
lgtm 🚀

**Pull request merged**